pythonforandroid/recipes/kivy3/__init__.py

```diff
@@ -0,0 +1,21 @@
+from pythonforandroid.recipe import PythonRecipe
+import shutil
+
+
+class Kivy3Recipe(PythonRecipe):
+    version = 'master'
+    url = 'https://github.com/KeyWeeUsr/kivy3/archive/{version}.zip'
```

Comment:
The kivy org is now owning the repository with more recent changes.
```suggestion
    url = 'https://github.com/kivy/kivy3/archive/{version}.zip'
```